packages/patternfly-4/react-charts/src/components/ChartArea/examples/ChartArea.md

```diff
@@ -85,16 +84,16 @@ import { Chart, ChartArea, ChartAxis, ChartGroup, ChartVoronoiContainer } from '
       </ChartGroup>
     </Chart>
   </div>
-</div>
+)
 ```
 
-```js title=Cyan-bottom-aligned-legend-and-axis-label
+```js title=Cyan-bottom--aligned-legend-and-axis-label
```

Comment:
This title should be "Cyan area chart with bottom aligned legend and axis label"

Also "bottom-aligned" does not need the dash. We did that for "bottom-left" since it matches the prop value